mlflow/types/llm.py

```diff
@@ -223,12 +223,13 @@ class ChatChoice(_BaseDataclass):
         index (int): The index of the response in the list of responses.
         message (:py:class:`ChatMessage`): The message that was generated.
         finish_reason (str): The reason why generation stopped.
+            **Optional**, defaults to ``"stop"``
         logprobs (:py:class:`ChatChoiceLogProbs`): Log probability information for the choice.
     """
 
     index: int
     message: ChatMessage
-    finish_reason: str
+    finish_reason: str = "stop"
```

Comment:
Q:

Is it risky to assume the default finish reason to be "stop" ?
sometimes the finish reason is lost but actual finish reason might be max-token limit or other reasons

Reply:
Yeah I think this is possible, though I think the intention of this PR is just to provide reasonable defaults when users don't have the info available. If they're integrating directly with openai or transformers, they can just put the actual finish reason in here.